kolibri/plugins/coach/assets/src/routes/examRoutes.js

```diff
@@ -52,8 +60,8 @@ export default [
     ],
   },
   {
-    name: QuizSummaryPage.name,
-    path: '/:classId/plan/quizzes/:quizId/:tabId?',
+    name: PageNames.EXAM_SUMMARY,
+    path: path(CLASS, QUIZ, '/:tabId?'),
```

Comment:
I am planning to expand this `path(...)` pattern, if there is any objections you can let me know :)

Reply:
I must admit, I don't particularly like it - but I can't quite put my finger on why (I didn't like its existing use in many different parts of the coach routing). I was hoping that we could use nested routes to do a lot of this sort of concatenation, but given the nature of the routing, I don't think we can.

So, I think this is fine - but I guess, is there any reason to not just do `CLASS + QUIZ + '/:tabId?'`?

Reply:
I dont liked it neither :laughing:, because for non-familiarized people its very tedious as we just cant inmediately see what is the actual path of each route. But for this refactor it was much easier to just remove the `/plan` on the CLASS constant, and I think it ensures the consistency as we will need to rewrite the same base routes a huge lot of times. I'm not opposed to reverting it, But I would like the entire coach module to be consistent in this regard.

> is there any reason to not just do CLASS + QUIZ + '/:tabId?'?
Not really, I can make this change.

Reply:
> would like to replicate in all the coach routes for consistency

for me this is the most important thing. 

I think my challenge with this existing routing pattern is that I find it a bit inconsistent within its own logic. For example in the reportsRoutes
`const CLASS = '/:classId/reports'` feels inconsistent `const QUIZ = '/quizzes/:quizId';`. I know it's necessary (it's the reports, not the plan tab), but it isn't really something you can logic out with normal API patterns when reading the code.

And that is not even including the differences in the files between reports and plan (which thankfully, will disappear with this refactor! 🎉) 

I wonder if there might be some small changes that we could make to have these feel internally more consistent, without too much scope creep? That might help with making them easier to read. I can try to come up with some more concrete suggestions if either of you think this is worth talking about more (or an example, if this feels vague). Some of this might be resolved just in the course of getting rid of the reports and plan differences, I just haven't thought it all through yet.

Reply:
I think maybe best to do the minimum for now in terms of the actual content of the routes, and then we can do smaller follow up issues for routes cleanup specifically.

Reply:
After migrating a huge amount of routes, I am glad we had these constants partial terms, it made everything much easier